casepro/statistics/signals.py

```diff
@@ -38,6 +41,29 @@ def record_new_outgoing(sender, instance, created, **kwargs):
         if instance.partner:
             DailyCount.record_item(day, DailyCount.TYPE_REPLIES, partner)
 
+        if case:
+            # count the very first response on an org level
+            if instance == case.outgoing_messages.earliest('created_on'):
```

Comment:
@rowanseymour I've checked the requirements on our side - we specifically need to now the average time from when a case was opened. For our use case, the national team will be triaging the messages and it's only when a case is opened and assigned to a provincial officer (partner) do we need to look at the responsiveness. Of particular interest we need to understand how responsive the provincial team is at dealing with cases.